mkdocs.yml

```diff
@@ -68,7 +68,7 @@ pages:
       - URL Groups: experience/url-groups.md
       - Configure Apdex: experience/configure-requirements.md
       - User Satisfaction: experience/user-satisfaction.md
-      - On-Page Transactions: experience/on-page-transaction.md
+      - Measurements: experience/measurements.md
```

Comment:
Should this be called "Performance Measurements" for a bit more clarity?

Reply:
I was going with something short that is in sync with how we show it in the UI, but I think we can make it more descriptive in the docs since there's space there.